docs/zh/latest/getting-started.md

```diff
@@ -1,249 +0,0 @@
-
-```bash
-curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
-```
-
-**响应内容：**
-
-```json
-{
-  "args": {
-    "foo1": "bar1",
-    "foo2": "bar2"
-  },
-  "headers": {
-    "Accept": "*/*",
-    "Host": "httpbin.org",
-    "User-Agent": "curl/7.29.0",
-    "X-Amzn-Trace-Id": "Root=1-6088fe84-24f39487166cce1f0e41efc9"
-  },
-  "origin": "58.152.81.42",
-  "url": "http://httpbin.org/get?foo1=bar1&foo2=bar2"
-}
-```
-
-## 安装 APISIX
-
-APISIX 可以借助 quickstart 脚本快速安装并启动。
-
-```sh
-curl -sL https://run.api7.ai/apisix/quickstart | sh
-```
-
-该命令在本地安装并运行了基于 Docker 的 APISIX 和 etcd 容器，其中 APISIX 采用 etcd 保存和同步配置信息。APISIX 和 etcd 容器使用 [**host**](https://docs.docker.com/network/host/) 的 Docker 网络模式，因此可以从本地直接访问。
-
-如果一切顺利，将输出如下信息。
-
-```text
-✔ APISIX is ready!
-```
-
-:::note
-
-你也可以参考 [APISIX 安装指南](./installation-guide.md)了解不同的安装方法。
-
-:::
-
-:::info IMPORTANT
-
-请确保其他系统进程没有占用 **9080、9180、9443 和 2379** 端口。
-
-:::
-
-你可以通过 curl 来访问正在运行的 APISIX 实例。比如，你可以发送一个简单的 HTTP 请求来验证 APISIX 运行状态是否正常。
-
-```sh
-curl "http://127.0.0.1:9080" --head | grep Server
-```
-
-如果一切顺利，将输出如下信息。
-
-```text
-Server: APISIX/Version
-```
-
-`Version` 是指您已经安装的 APISIX 的版本。例如，`APISIX/3.3.0`。
-
-现在，你已经成功安装并运行了 APISIX！
-
-## 创建路由
-
-APISIX 提供了强大的 [Admin API](./admin-api.md) 和 [Dashboard](https://github.com/apache/apisix-dashboard) 供用户使用。在下述示例中，我们将使用 Admin API 创建一个 [Route](./terminology/route.md) 并与 [Upstream](./terminology/upstream.md) 绑定，当一个请求到达 APISIX 时，APISIX 会将请求转发到指定的上游服务中。
-
-以下示例代码中，我们将为路由配置匹配规则，以便 APISIX 可以将请求转发到对应的上游服务：
-
-```bash
-curl "http://127.0.0.1:9180/apisix/admin/routes/1" -X PUT -d '
-{
-  "methods": ["GET"],
-  "host": "example.com",
-  "uri": "/anything/*",
-  "upstream": {
-    "type": "roundrobin",
-    "nodes": {
-      "httpbin.org:80": 1
-    }
-  }
-}'
-```
-
-该配置意味着，当请求满足下述的**所有**规则时，请求将被转发到上游服务（`httpbin.org:80`）：
-
-- 请求的 HTTP 方法为 `GET`。
-- 请求头包含 `host` 字段，且它的值为 `example.com`。
-- 请求路径匹配 `/anything/*`，`*` 意味着任意的子路径，例如 `/anything/foo?arg=10`。
-
-当路由创建完成后，可以通过以下命令访问上游服务：
-
-```bash
-curl -i -X GET "http://127.0.0.1:9080/anything/foo?arg=10" -H "Host: example.com"
-```
-
-该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/foo?arg=10`。
-
-## 使用上游服务创建路由
-
-你可以通过以下命令创建一个上游，并在路由中使用它，而不是直接将其配置在路由中：
-
-```bash
-curl "http://127.0.0.1:9180/apisix/admin/upstreams/1" -X PUT -d '
-{
-  "type": "roundrobin",
-  "nodes": {
-    "httpbin.org:80": 1
-  }
-}'
-```
-
-该上游配置与上一节配置在路由中的上游相同。同样使用了 `roundrobin` 作为负载均衡机制，并设置了 `httpbin.org:80` 为上游服务。为了将该上游绑定到路由，此处需要把 `upstream_id` 设置为 `"1"`。更多字段信息，请参考 [Admin API](./admin-api.md)。
-
-上游服务创建完成后，可以通过以下命令绑定到指定路由：
-
-```bash
-curl "http://127.0.0.1:9180/apisix/admin/routes/1" -X PUT -d '
-{
-  "methods": ["GET"],
-  "host": "example.com",
-  "uri": "/anything/*",
-  "upstream_id": "1"
-}'
-```
-
-我们已经创建了路由与上游服务，现在可以通过以下命令访问上游服务：
-
-```bash
-curl -i -X GET "http://127.0.0.1:9080/anything/foo?arg=10" -H "Host: example.com"
-```
-
-该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/foo?arg=10`。
-
-## 使用 APISIX Dashboard
```

Comment:
Not sure if we should remove this doc completely. The getting start docs built by API7 would not mention anything about dashboard.

Reply:
Can we add apisix-dashboard.md in /getting-started?